website/source/guides/operations/reference-architecture.html.md

```diff
@@ -237,10 +240,28 @@ data such as ephemeral authentication tokens, time-based token information as
 well as token usage data. This provides for aggressive recovery point objective
 in environments where high availability is of the utmost concern.
 
-#### Diagrams
+#### Cross-Region Disaster Recovery
+
+If your disaster recovery strategy is to plan for a loss of an entire data
+center, the following diagram illustrates a possible replication scenario.
+
+![Replication Pattern](/assets/images/vault-ref-arch-4.png)
+
+In this scenario, if the Vault cluster in Region A fails over to a DR cluster in
```

Comment:
Maybe instead of "if the Vault cluster in Region A fails over to a DR cluster in Region B" we say something along the lines of "if the Vault cluster in Region A fails and you promote the DR cluster in Region B"? The cluster doesn't necessarily failover, it just fails and the DR in Region B would be promoted. The apps would need failover logic, but Vault doesn't have any.